ios-sdk/TangramVC.swift

```diff
@@ -12,6 +12,7 @@ class TangramVC:  TGMapViewController{
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        self.queueSceneUpdate("sources.mapzen.url_params.api_key", withValue: "vector-tiles-7bfyfz8")
```

Comment:
I see, previously this was hardcoded, it has been removed since a while back from `TGMapController` within tangram-es. We don't want the user to load that `scene.yaml` by default (https://github.com/tangrams/tangram-es/commit/97062b84150db5736b347492f8e9426824c6b8bb#diff-6e7af98500d6b23bde5e916d3e8dc4f3L353) but instead use `loadScene` explicitely, so we don't have to carry on that `scene.yaml` resource while distributing the framework.

As for the `applyUpdates()`, it's definitely missing from the interface. Pushing a short fix shortly!